Monika After Story/game/zz_consumables.rpy

```diff
@@ -1432,7 +1432,8 @@ init 5 python:
 
 label mas_consumables_generic_queued_running_out:
     $ low_cons = MASConsumable._getLowCons()
-    call mas_consumables_generic_queued_running_out_dlg(low_cons)
+    if low_cons:
```

Comment:
Ideally, wouldn't this be better to put in the `mas_consumables_generic_queued_running_out_dlg` label?

Since we know that `low_cons` would be an empty list, so checking in that label if the passed in list is empty would suffice and save repeat code